### ISSUE TITLE
Gallop rhythm

### ISSUE THREAD

**Issue by tudorgroza** [obophenotype/human-phenotype-ontology#6088]
**Preferred term label:**
Gallop rhythm

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
HP:0011675?

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Are there actually clinicians who still appreciate this in these tech-filled days? I'm impressed!

**Comment by pnrobinson** [comment]
Added as HP:0033113
